### PR TITLE
[GITHUB-3] Switched to Pip for install

### PR DESCRIPTION
**Pull request by lobsterdore**
Adds installation procedure taken from OS packages so we can use
Pip to install a range of versions. Mimics Debian package install
by adding config files and log directories along with a SysVinit
script.

Also adds option to disable automatic start on boot, this will
allow Supervisor to be started only once all config is in place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +70,41 @@ To add some config settings (see [http://supervisord.org/configuration.html]()):
...
+
+    - name: Ensure supervisor service is started
+      become: yes
+      service:
+        name: supervisor
+        state: started
+
+    - name: Ensure service is present
+      become: yes
+      supervisorctl:
```
Comment (dennisconrad) [inline README.md]: Should this and below be

```yaml
service:
  name: supervisorctl
```
?

Comment (lobsterdore) [reply README.md]: I am ensuring that a supervisor controlled service is present, hence the use the of the supervisorctl module.

**Review by dennisconrad — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (dennisconrad) [reply README.md]: Just making sure because of the `name: service`.  Approved.

Comment (lobsterdore) [reply README.md]: Renamed to my_app since it was a bit confusing.

**Review by rsgoodall — APPROVED** [review]

**Pull request merged**